### PR TITLE
fixed v16 internal links

### LINKED ISSUE #1170: Broken internal links - v16
Part of the https://github.com/vitessio/website/issues/1169 umbrella to fix broken links

Below is a truncated output of broken internal links -- filtered for v16 results.

```
➜  vitess.io git:(prod) make check-internal-links
rm -rf public
npm install

up to date, audited 176 packages in 941ms

40 packages are looking for funding
  run `npm fund` for details

found 0 vulnerabilities
npx hugo --cleanDestinationDir -e dev -DFE
WARN 2022/10/12 10:24:31 Unknown Sitemap field: defaultcontentlanguage
WARN 2022/10/12 10:24:31 Unknown Sitemap field: defaultcontentlanguageinsubdir
WARN 2022/10/12 10:24:31 Unknown Sitemap field: disablelanguages
WARN 2022/10/12 10:24:31 Unknown Sitemap field: defaultcontentlanguage
WARN 2022/10/12 10:24:31 Unknown Sitemap field: defaultcontentlanguageinsubdir
WARN 2022/10/12 10:24:31 Unknown Sitemap field: disablelanguages
Start building sites …
hugo v0.96.0-2fd4a7d3d6845e75f8b8ae3a2a7bd91438967bbb+extended darwin/arm64 BuildDate=2022-03-26T09:15:58Z VendorInfo=gohugoio

                   |  EN  | ZH
-------------------+------+------
  Pages            | 1664 | 396
  Paginator pages  |    0 |   0
  Non-page files   |  108 |  12
  Static files     |  120 | 120
  Processed images |    0 |   0
  Aliases          |  734 |   0
  Sitemaps         |    2 |   1
  Cleaned          |    0 |   0

Total in 2502 ms
curl https://raw.githubusercontent.com/wjdp/htmltest/master/godownloader.sh | bash
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed
100 10278  100 10278    0     0  57657      0 --:--:-- --:--:-- --:--:-- 59755
wjdp/htmltest info checking GitHub for latest tag
wjdp/htmltest info found version: 0.16.0 for v0.16.0/macos/arm64
wjdp/htmltest info installed ./bin/htmltest
bin/htmltest
htmltest started at 10:24:37 on public/docs
========================================================================

⋮

16.0/user-guides/vschema-guide/backfill-vindexes/index.html
  hash does not exist --- 16.0/user-guides/vschema-guide/backfill-vindexes/index.html --> #CreateLookupVindex
16.0/user-guides/vschema-guide/subsharding-vindex/index.html
  target does not exist --- 16.0/user-guides/vschema-guide/subsharding-vindex/index.html --> ../../../reference/compatibility/query_planner
16.0/user-guides/schema-changes/managed-online-schema-changes/index.html
  target does not exist --- 16.0/user-guides/schema-changes/managed-online-schema-changes/index.html --> ../../../design-docs/table-lifecycle/safe-lazy-drop-tables/
  target does not exist --- 16.0/user-guides/schema-changes/managed-online-schema-changes/index.html --> ../../../design-docs/online-ddl/scheduler
16.0/user-guides/schema-changes/recoverable-migrations/index.html
  target does not exist --- 16.0/user-guides/schema-changes/recoverable-migrations/index.html --> ../../../design-docs/online-ddl/scheduler
16.0/user-guides/sql/vtexplain-in-bulk/index.html
  target does not exist --- 16.0/user-guides/sql/vtexplain-in-bulk/index.html --> ../../../contributing
16.0/user-guides/sql/vtexplain/index.html
  target does not exist --- 16.0/user-guides/sql/vtexplain/index.html --> ./explain-format-vtexplain
  target does not exist --- 16.0/user-guides/sql/vtexplain/index.html --> ../../../contributing
16.0/reference/vreplication/metrics/index.html
  hash does not exist --- 16.0/reference/vreplication/metrics/index.html --> ../../features/monitoring/#3-push-based-metrics-system#3-push-based-metrics-system
16.0/reference/vreplication/flags/index.html
  target does not exist --- 16.0/reference/vreplication/flags/index.html --> ../../../design-docs/vreplication/vstream/tracker/
16.0/reference/vreplication/progress/index.html
  target does not exist --- 16.0/reference/vreplication/progress/index.html --> ../../../design-docs/vreplication/life-of-a-stream
16.0/reference/vreplication/movetables/index.html
  target does not exist --- 16.0/reference/vreplication/movetables/index.html --> ../../../../../docs/user-guides/configuration-advanced/unmanaged-tablet/#move-legacytable-to-the-commerce-keyspace
  target does not exist --- 16.0/reference/vreplication/movetables/index.html --> ../../../../../docs/user-guides/migration/move-tables/
  target does not exist --- 16.0/reference/vreplication/movetables/index.html --> ../../../../../docs/concepts/move-tables/
16.0/reference/features/schema-management/index.html
  hash does not exist --- 16.0/reference/features/schema-management/index.html --> ../../../reference/programs/vtctl/#getschema
  hash does not exist --- 16.0/reference/features/schema-management/index.html --> ../../../reference/programs/vtctl/#listalltablets
  hash does not exist --- 16.0/reference/features/schema-management/index.html --> ../../../reference/programs/vtctl/#validateschemashard
  hash does not exist --- 16.0/reference/features/schema-management/index.html --> ../../../reference/programs/vtctl/#validateschemakeyspace
  hash does not exist --- 16.0/reference/features/schema-management/index.html --> ../../../reference/programs/vtctl/#getvschema
  hash does not exist --- 16.0/reference/features/schema-management/index.html --> ../../../reference/programs/vtctl/#getsrvvschema
  hash does not exist --- 16.0/reference/features/schema-management/index.html --> ../../../reference/programs/vtctl/#applyvschema
  hash does not exist --- 16.0/reference/features/schema-management/index.html --> ../../../reference/programs/vtctl/#applyvschema
  hash does not exist --- 16.0/reference/features/schema-management/index.html --> ../../../reference/programs/vtctl/#rebuildvschemagraph
16.0/reference/features/mysql-replication/index.html
  hash does not exist --- 16.0/reference/features/mysql-replication/index.html --> ../../programs/vtctl/#emergencyreparentshard
16.0/reference/features/vindexes/index.html
  target does not exist --- 16.0/reference/features/vindexes/index.html --> ../../../design-docs/query-serving/clookup-vindex/
16.0/reference/programs/vtctl/index.html
  hash does not exist --- 16.0/reference/programs/vtctl/index.html --> ../vtctl/keyspaces#removekeyspacesell
  hash does not exist --- 16.0/reference/programs/vtctl/index.html --> ../vtctl/keyspaces#createLookupvindex
  hash does not exist --- 16.0/reference/programs/vtctl/index.html --> ../vtctl/keyspaces#VDiff
  hash does not exist --- 16.0/reference/programs/vtctl/index.html --> ../vtctl/serving-graph#getsrvsvchema
16.0/reference/vtctldclient-transition/overview/index.html
  hash does not exist --- 16.0/reference/vtctldclient-transition/overview/index.html --> ../../programs/vtctld/#grpc-vtctld-mdash-new-in-v14
16.0/get-started/index.html
  target does not exist --- 16.0/get-started/index.html --> ../contributing/build-on-centos
  target does not exist --- 16.0/get-started/index.html --> ../contributing/build-on-macos
  target does not exist --- 16.0/get-started/index.html --> ../contributing/build-on-ubuntu
16.0/get-started/local-brew/index.html
  target does not exist --- 16.0/get-started/local-brew/index.html --> ../../../overview/supported-databases/
16.0/get-started/local-mac/index.html
  target does not exist --- 16.0/get-started/local-mac/index.html --> ../../../overview/supported-databases/

⋮

========================================================================
✘✘✘ failed in 1.056837833s
270 errors in 1571 documents
make: *** [run-link-checker] Error 1
```

### PR DESCRIPTION
**Pull request by PaarthAgarwal**
Fixed v16 internal links and target hashes. Partially fixes #1170.

### CODE REVIEW

**Comment by nate-double-u** [comment]
Thanks for this @PaarthAgarwal, please sign the DCO -- this repository requires each commit to be signed before we merge. You can find the instructions here: https://github.com/vitessio/website/pull/1360/checks?check_run_id=10883304143

**Comment by nate-double-u** [comment]
I've approved the preview, so you can see the preview here: https://deploy-preview-1360--vitess.netlify.app/

**Comment by PaarthAgarwal** [comment]
Finally signed off. Had to regenerate my personal access token but worth it haha

File: content/en/docs/16.0/reference/features/vindexes.md [hunk 1]
```diff
@@ -64,7 +64,7 @@ The lookup table that implements a Lookup Vindex can be sharded or unsharded.  N
...
 
-Consistent lookup vindexes use an alternate approach that makes use of careful locking and transaction sequences to guarantee consistency without using 2PC. This gives the best of both worlds, with the benefit of a consistent cross-shard vindex without paying the price of 2PC. To read more about what makes a consistent lookup vindex different from a standard lookup vindex read our [consistent lookup vindexes design documentation](../../../../design-docs/query-serving/clookup-vindex).
+Consistent lookup vindexes use an alternate approach that makes use of careful locking and transaction sequences to guarantee consistency without using 2PC. This gives the best of both worlds, with the benefit of a consistent cross-shard vindex without paying the price of 2PC. To read more about what makes a consistent lookup vindex different from a standard lookup vindex read our [consistent lookup vindexes design documentation](../../../../design-docs/query-serving).
```
Comment (frouioui) [inline content/en/docs/16.0/reference/features/vindexes.md]: The link to consistent lookup vindex design documentation should be: https://github.com/vitessio/vitess/issues/4855.

**Review by frouioui — DISMISSED** [review]

**Comment by nate-double-u** [comment]
> Finally signed off. Had to regenerate my personal access token but worth it haha

Thanks for doing that! Should be easier now for your next PRs 🙂

File: content/en/docs/16.0/reference/features/vindexes.md (hunk 1 above)
Comment (PaarthAgarwal) [reply content/en/docs/16.0/reference/features/vindexes.md]: Done!

**Review by mattlord — APPROVED** [review]
Thanks again, @PaarthAgarwal ! ❤️

**Pull request merged**